commands/build/lib/init-config.js

```diff
@@ -40,22 +39,44 @@ const options = {
   },
 };
 
+// nebula build --watch                - watch umd bundle
+// nebula build --watch umd            - watch umd bundle
+// nebula build --watch systemjs       - watch systemjs bundle
+// nebula build                        - watch disabled
+const watchMiddleware = (argv) => {
```

Comment:
@T-Wizard Ended up using a middleware to get the desired behaviour. Found the [coerce](http://yargs.js.org/docs/#api-reference-coercekey-fn) setting in yargs which would fit nicely with the use case at hand but it turns out that it doesn't work properly with our setup. The options seems to initialised twice and coerce is only run in the first execution.

Reply:
An alternate solution to this is to use two options. `watch` controls if watch is enabled and `format` controls what to watch for.

```js
nebula build --watch                     - watch umd bundle
nebula build --watch --format umd        - watch umd bundle
nebula build --watch --format systemjs   - watch umd bundle
```

It's more verbose but to me a bit more explicit.

Reply:
It looks good to me!